src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -122,7 +122,7 @@ object Helpers extends SparkSessionWrapper {
    * TODO: rename to defaultParallelism
    */
   private def parallelism: Int = {
-    Math.min(driverCores, 8)
+    Math.max(driverCores * 2, 8)
```

Comment:
we should be careful with running parallel optimize, because it may overload cluster. VACUUM is ok with parallelization

Reply:
good call, I will reduce parallelize with optimize. I have it set in a way so as not to overload it but I will make cluster-size recommendations in the docs for this. Optimize is generally safe at driverCores - 1 from my experience.